example/areaLight.js

```diff
@@ -0,0 +1,268 @@
+import * as THREE from 'three';
+import { FullScreenQuad } from 'three/examples/jsm/postprocessing/Pass.js';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+import { PathTracingRenderer, PhysicalPathTracingMaterial } from '../src/index.js';
+import { PathTracingSceneWorker } from '../src/workers/PathTracingSceneWorker.js';
+import { RGBELoader } from 'three/examples/jsm/loaders/RGBELoader.js';
+import { GUI } from 'three/examples/jsm/libs/lil-gui.module.min.js';
+import { TransformControls } from 'three/examples/jsm/controls/TransformControls.js';
+import { Scene } from 'three';
+
+let renderer, controls, transformControls, transformControlsScene, areaLights, sceneInfo, ptRenderer, camera, fsQuad;
+let samplesEl;
+const params = {
+
+	environmentIntensity: 0.5,
+	environmentRotation: 0,
+	emissiveIntensity: 100,
+	bounces: 20,
+	samplesPerFrame: 1,
+	resolutionScale: 1 / window.devicePixelRatio,
+	filterGlossyFactor: 0.25,
+	tiles: 1,
+	multipleImportanceSampling: true
+
+};
+
+// clamp value for mobile
+const aspectRatio = window.innerWidth / window.innerHeight;
+if ( aspectRatio < 0.65 ) {
+
+	params.bounces = Math.min( params.bounces, 10 );
+	params.resolutionScale *= 0.5;
+	params.tiles = 3;
+
+}
+
+init();
+
+async function init() {
+
+	renderer = new THREE.WebGLRenderer( { antialias: true } );
+	renderer.toneMapping = THREE.ACESFilmicToneMapping;
```

Comment:
Related to #185 - I'd forgotten to include the line to set the output color space to sRGB in the other examples, so we should add the following here, as well:

`renderer.outputEncoding = THREE.sRGBEncoding`